lips/lip-6.md

```diff
@@ -0,0 +1,88 @@
+If a transaction causes an epoch change, that transaction is the last transaction committed
+in the current epoch — any subsequent transactions in that block or future blocks from that epoch
+will be ignored. Once the transaction has been committed, the new validator set can start the next epoch of the consensus protocol.
+
+Only LibraRoot may add/remove validators owner accounts to/from the validator set. Each validator operator may update its validator's config in this set which will also trigger the LibraConfig::NewEpochEvent.
+
+Each validator owner is obliged to notify LN about the decision to change the validator operator, LN may disallow the switch if the new operator is deemed not trustworthy or the distribution of consensus power becomes skewed towards this operator or operators with similar architectures. LN makes sure there is enough diversity between the consensus nodes of the Libra Blockchain.
+
+# Transaction Scripts
+The following transaction scripts can be run by the LibraRoot:
+* `create_validator_account`
+* `create_validator_operator_account`
+* `add_validator_and_reconfigure`
+* `remove_validator_and_reconfigure`
+
+The following transaction scripts can be run by the validator owner:
+* `set_validator_operator`
+* `remove_validator_operator`
+* `rotate_authentication_key`
+
+The following transaction scripts can be run by the validator operator:
+* `register_validator_config`
+* `set_validator_config_and_reconfigure`
+* `rotate_authentication_key`
+
+# Notes
+In first version of Libra Protocol, validator owners will not maintain ownership of their Libra accounts, the LibraRoot account will run validator owners' operations on their behalf. Validator operators will maintain full ownership of their Libra accounts.
+Gradually, the validator owners will start managing their accounts themselves. They will send their new account key to the Association and the Association will rotate their account key to this new key. Right after this rotation transaction gets executed, the validator owner starts to maintain its Libra account.
+
+Both the validator owner account and the validator operator account store a human_name which refers to the name of the company behind the account written in snake case (lowercase, where spaces are replaced with underscores), it is initialized at account creation and never changes. Most validator-related transaction scripts get the address and the names of the account and check that the human_name under the account matches the name passed into the transaction script. This prevents accounts from accidentally mix up the addresses, which are hex strings, and adds a second level of protection against human errors.
+
+### Formal verification guarantees on the Move Implementation
+
+* Disciplined aborts
+* Only LibraRoot or a Validator Operator can trigger reconfiguration
+* Every change to an validator set triggers a reconfiguration
+* Only a validator can set its operator
```

Comment:
this contradicts the note above

Reply:
This refers to how the module is written, the fact that LibraRoot is calling it via an admin script (which is essentially a simpler version of the write-set) is a temporary measure. There is always a possibility that the write-set transaction changes the blockchain in some unintended way... What would you suggest writing here?

Reply:
I see. This makes sense. 

How about : Only Validator Owner (role) can set an operator; for each Validator Owner account, this permission is given to only one Validator Owner (itself).

(as a side note, as mentioned above, an admin script by LibraRoot may act on behalf of Validator Owner accounts.)